### DISCUSSION TITLE
Unable to get Verification Email

### DISCUSSION THREAD

**Discussion by Maude-Mayhem** [mastodon/mastodon#20124] [Q&A] [answered]
Hi, My name is Maude and I have tried to sign up to Mastodon with two different emails half an hour ago. i still have yet to receive any of the verification emails. I have checked in my spam boxes. they are not there. i have changed the address but still nothing. I apologize if this is an error on my end. the email for this account came through relatively well too.

**Comment by iUser486** [comment]
> Hi, My name is Maude and I have tried to sign up to Mastodon with two different emails half an hour ago. i still have yet to receive any of the verification emails. I have checked in my spam boxes. they are not there. i have changed the address but still nothing. I apologize if this is an error on my end. the email for this account came through relatively well too.

Having the same issue here.  Tried two different servers and several different email accounts.  No joy.

**Reply by marekyarbrough** [reply]
Se problem.

**Reply by Adagio4639** [reply]
Same issue Maude. And it's now December 16. It appears that this platform has some unresolved problems. It recognizes my username, as I post this, but I can't login because there's no confirmation email that confirms my account.

**Reply by mohe2015** [reply]
As Mastodon is *federated* it matters on which server/website you registered. It likely is an issue only the concrete server operator can fix. you can usually find their contact information on the about page of the server's website.

**Comment by Valsisttv** [comment]
I'm also on the same boat and without a verification email since I made my account.

I even emailed them and got no reply what so ever, nor through Twitter DM.

**Reply by mohe2015** [reply]
As Mastodon is federated you can always choose another server instance. Maybe that server's operator got too many new members or had other reasons not to answer.

**Comment by gymshorts2tight** [comment]
I am having this problem as well. Requested multiple times to have it resent, but it isn't appearing in any of my folders or through any filters. Tried to use the "Hide My Email" feature/option my phone gave me, but the email never sent through that either. So, now I have two accounts, can't get onto one of them, and the other isn't getting any confirm emails. If I can't get a fix for this soon, I'm out of here. No reason why it should take this long to send a fucking confirmation email, or why it has to be this complicated.

**Reply by Adagio4639** [reply]
Exactly. I heard that this place was an alternative to Twitter, but this problem in simply creating an account is stupid. I click on the send confirmation email and I get nothing and now I see that this has been an ongoing issue for the past month, and it's still not fixed. And apparently nobody bothers to check this forum to even see that there is a problem. I have no intentions of sticking around here forever. Get the problem fixed. This should be basic to getting your thing off the ground.

**Reply by mohe2015** [reply]
See my answers on the other questions in this thread.

**Comment by moench4333** [comment]
I have the same problem as well. I get an email, but when i click on it, the profile opens with the message that the email address is not yet registered :-(

**Reply by mohe2015** [reply]
See my answers on the other questions in this thread. If that doesnt resolve it maybe you can specify the instance you registered on.

**Comment by nightpool** [comment, marked as answer]
This forum is for the development and maintenance of the Mastodon server software. This is a free software package that can be run by anyone who wants to provide their own, connected, federated social media software. It is not a website, just code that others use to run a website.

The people who maintain this software are, for the most part, not involved or associated with the individual websites that people run or sign up on. You'll have to contact the person running your server to get more details about what may or may not be happening here.
